docs/getting-started/advanced-setting.asciidoc

```diff
@@ -76,6 +76,7 @@ IMPORTANT: Threat intelligence indices aren't required to be ECS-compatible for
 
 Kibana transmits certain information about Elastic Security when users interact with the {security-app}, detailed below. {kib} redacts or obfuscates personal data (IP addresses, host names, usernames, etc.) before transmitting messages to Elastic. Security-specific telemetry events include:
 
+* *{detection-rule} Security alerts:* Information about Elastic authored detection rules using the detection engine. Examples of alert data include Machine Learning job influencers, process names, and cloud audit events.
```

Comment:
We don't have an attribute defined for `{detection-rule}` so that can just be hard-written. A few other minor edits:
```suggestion
* *Detection rule security alerts:* Information about Elastic-authored prebuilt detection rules using the detection engine. Examples of alert data include machine learning job influencers, process names, and cloud audit events.
```

Reply:
Thanks, @joepeeples! Appreciated :)